MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -29,7 +29,7 @@
 #import "MatrixSDKSwiftHeader.h"
 #import "MXFileRoomSummaryStore.h"
 
-static NSUInteger const kMXFileVersion = 79;
+static NSUInteger const kMXFileVersion = 80;
```

Comment:
As I understand this, incrementing file store version will delete all data and trigger initial sync. This will ensure all state files with already corrupted data is removed and we start monitoring for empty room state from this point onwards. Are there any further consequences for when this gets released to production users?